modules/clair-crda-configuration.adoc

```diff
@@ -0,0 +1,43 @@
+[[clair-crda-configuration]]
+= Clair CRDA configuration
+
+== Enabling Clair CRDA
+
+With {productname} 3.7, Java scanning no longer includes a default CRDA shared key, and is no longer enabled by default. Use the following procedure to fetch the Quay-specific CRDA remote matcher, which supports higher RPS, and to manually enable CRDA for Java scanning. 
+
+.Prerequisites
+
+* {productname} 3.7
+
+.Procedure
+
+. To fetch the Quay-specific CRDA remote matcher, `GET` the data from `https://gw.api.openshift.io/api/v2/get-token?user_key=207c527cfc2a6b8dcf4fa43ad7a976da`:
++
+[source,terminal]
+----
+$ curl --location --request GET 'https://gw.api.openshift.io/api/v2/get-token?user_key=207c527cfc2a6b8dcf4fa43ad7a976da'
```

Comment:
Running this multiple times gives the same value back, the one already noted below. This seems wrong; someone should look into that.